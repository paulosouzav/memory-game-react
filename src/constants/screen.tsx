interface Indexable {
  [key: string]: number;
}

export const MAX_WIDTHS: Indexable = Object.freeze({
  sm: 540,
  md: 720,
  lg: 960,
  xl: 1140,
});

export const BREAKPOINTS: Indexable = Object.freeze({
  sm: 576,
  md: 758,
  lg: 992,
  xl: 1200,
});
