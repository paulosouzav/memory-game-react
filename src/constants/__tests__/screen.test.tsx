import { BREAKPOINTS, MAX_WIDTHS } from '../screen';

describe('BREAKPOINTS', () => {
  it('should have expected values', () => {
    expect(BREAKPOINTS).toMatchSnapshot();
  });
});

describe('MAX_WIDTHS', () => {
  it('should have expected values', () => {
    expect(MAX_WIDTHS).toMatchSnapshot();
  });
});
