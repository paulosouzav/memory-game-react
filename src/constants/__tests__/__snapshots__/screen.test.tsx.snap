// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BREAKPOINTS should have expected values 1`] = `
Object {
  "lg": 992,
  "md": 758,
  "sm": 576,
  "xl": 1200,
}
`;

exports[`MAX_WIDTHS should have expected values 1`] = `
Object {
  "lg": 960,
  "md": 720,
  "sm": 540,
  "xl": 1140,
}
`;
